mk_verificator/benchmarks/scenarios/dd_tests/test_dd.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import time
+import mk_verificator.utils as utils
+import mk_verificator.utils.ssh_client as ssh
+import mk_verificator.fixtures.base
+# TODO
+# Create fixture for adding/removing keypair
+# Create fixture for checking security groups
+
+
+@pytest.mark.parametrize(
+    'count,block_size',
+    [('4k', 262144), ('1M', 1024), ('1G', 1)]
```

Comment:
please add more pairs here, you can get all combinations of multiple parametrized arguments, just stack parametrize decorators:
```
@pytest.mark.parametrize("x", [0, 1])
@pytest.mark.parametrize("y", [2, 3])
def test_foo(x, y):
    pass
```